reference/env_vars.rst

```diff
@@ -344,3 +344,30 @@ uploading packages, as they will be read-only and that could have other side-eff
 
     It is not recommended to upload packages directly from developers machines with read-only mode as it could lead to insconsistencies.
     For better reproducibility we recommend that packages are created and uploaded by CI machines.
+
+.. _conan_run_tests:
+
+CONAN_RUN_TESTS
```

Comment:
Please add an example here with tools.get_env()

Reply:
@memsharded  going to add a reference to ``tools.get_env()`` which already has examples on how to use it, if you are ok

Reply:
I am not thinking of a ``tools.get()`` example, but a full ``build()`` example, which conditionally executes ``cmake.test()`` depending on this variable. Better to make it very explicit what is the typical usage of this env-var.

Reply:
@memsharded see https://github.com/conan-io/docs/pull/515/files#diff-a8f9db20cc1b4363089fee60bb3d9cc1R376

Reply:
Nop, as I said before, the full ``build()``:

```python
        def build(self): # this is not building a library, just tests
             cmake = CMake(self)
             cmake.configure()
             cmake.build()
             if tools.get_env("CONAN_RUN_TESTS", True):
                 cmake.test()
```
This is generic, both for header only and not header only. No need to specialize the example for header-only, or copy the full recipe.

Reply:
@memsharded fixed, check it